client/landing/gutenboarding/path.ts

```diff
@@ -116,6 +116,13 @@ export function useIsAnchorFm(): boolean {
 	return Boolean( anchorFmPodcastId && anchorFmPodcastId.match( /^[0-9a-f]{7,8}$/i ) );
 }
 
+export function useOnboardingFlow(): string {
```

Comment:
I'd argue for just getting rid of the constants, since we have one place returning the value, but this is totally fine and solves the problem to move forward – let's do it!

Reply:
@kwight I'll remove it the next time I touch this again, so we don't have to go through the approval dance